File: outpatient-service/src/main/java/com/medconsult/outpatient/department/entity/Department.java
package com.medconsult.outpatient.department.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.medconsult.common.mybatis.BaseEntity;
import lombok.Getter;
import lombok.Setter;

/**
 * 科室表（对应《数据库设计文档》§2.2 department 表）。
 *
 * <p>维护医院科室基础信息，支撑医生管理、排班和智能分诊推荐
 * （《需求文档》§4.3.1）。本服务对科室只读查询，创建/维护接口暂未实现。
 */
@Getter
@Setter
@TableName("department")
public class Department extends BaseEntity {

    /** 科室编号，如 DEP_CARDIOLOGY（业务可读，对外暴露） */
    private String departmentNo;

    /** 科室名称 */
    private String name;

    /** 科室介绍 */
    private String description;

    /** 科室位置 */
    private String location;

    /** 是否启用：1 启用 0 停用 */
    private Integer enabled;
}

File: src/mock/department.js
// 科室列表（对齐 department 表字段）
export const mockDepartmentList = () => {
  return {
    code: 0,
    message: 'success',
    data: [
      { id: 1, departmentNo: 'DEP_CARDIOLOGY', name: '心血管内科', description: '高血压、冠心病、心律失常', location: '门诊楼3层', enabled: 1 },
      { id: 2, departmentNo: 'DEP_RESPIRATORY', name: '呼吸内科', description: '肺炎、哮喘、慢性支气管炎', location: '门诊楼3层', enabled: 1 },
      { id: 3, departmentNo: 'DEP_GASTRO', name: '消化内科', description: '胃炎、胃溃疡、肝病', location: '门诊楼4层', enabled: 1 },
      { id: 4, departmentNo: 'DEP_ORTHOPEDICS', name: '骨科', description: '骨折、颈椎病、腰椎病', location: '门诊楼5层', enabled: 1 },
      { id: 5, departmentNo: 'DEP_PEDIATRICS', name: '儿科', description: '儿童常见病、生长发育', location: '门诊楼2层', enabled: 1 },
      { id: 6, departmentNo: 'DEP_DERMATOLOGY', name: '皮肤科', description: '皮炎、湿疹、痤疮', location: '门诊楼4层', enabled: 1 }
    ]
  }
}